tensorboard/webapp/widgets/card_fob/card_fob_controller_component.ts

```diff
@@ -146,10 +146,12 @@ export class CardFobControllerComponent {
     newStep: number,
     timeSelection: TimeSelection
   ): TimeSelection {
+    let newTimeSelection = {start: timeSelection.start, end: timeSelection.end};
```

Comment:
I'm super happy to see this cleanup.
I think it makes more sense to remove the argument and just reference the property directly. Also you never reassign `newTimeSelection` so it can be `const`
```suggestion
    const newTimeSelection = {...this.timeSelection};
```

Reply:
Done.